uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/manage-leave-requests.js

```diff
@@ -53,6 +57,7 @@ define([
     vm.isFilterExpanded = false;
     // leaveRequests.table - to handle table data
     // leaveRequests.filter - to handle left nav filter data
+    vm.isAdmin = false; // this property is updated on controller initialization
```

Comment:
```
// leaveRequests.table - to handle table data
// leaveRequests.filter - to handle left nav filter data
```
These comments are meant for `vm.leaveRequests`, please keep that order. Also alphabetically `isAdmin` should go before `isFilterExpanded`.
 